app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1261,10 +1261,7 @@ def test_rollout_invalid_version_support(self):
 
         self.assertFalse(serializer.is_valid())
         self.assertEqual(len(serializer.errors), 1)
-        self.assertEqual(
-            serializer.errors["is_rollout"][0],
-            NimbusConstants.ERROR_ROLLOUT_VERSION_SUPPORT,
-        )
+        self.assertIn("is_rollout", serializer.errors)
```

Comment:
Suggest adding a test that checks for the version number in the error message.

Reply:
thanks @mikewilli  will add later in the enhancement